jbpm-services/jbpm-kie-services/src/main/resources/META-INF/Servicesorm.xml

```diff
@@ -1383,76 +1107,351 @@
+                  t.workItemId AS t_workItemId,
+                  t.formName,
+                  t.subject,
+                  a.taskId,
+                  a.status AS a_status,
+                  a.activationTime AS a_activationTime,
+                  a.name AS a_name,
+                  a.description AS a_description,
+                  a.priority AS a_priority,
+                  a.actualOwner,
+                  a.createdBy,
+                  a.deploymentId AS a_deploymentId ,
+                  a.processId AS a_processId,
+                  a.processInstanceId AS a_processInstanceId,
+                  a.createdOn AS a_createdOn,
+                  a.dueDate,
+                  a.workItemId AS a_workItemId
+           FROM AuditTaskImpl a
+           LEFT OUTER JOIN Task t ON a.taskId = t.id
+           WHERE a.processInstanceId = :processInstanceId and a.status in ( :statuses)
+           UNION SELECT t.id,
+                        t.status,
+                        t.activationTime,
+                        t.name,
+                        t.description,
+                        t.priority,
+                        t.actualowner_id,
+                        t.createdby_id,
+                        t.deploymentId,
+                        t.processId,
+                        t.processInstanceId,
+                        t.createdOn,
+                        t.expirationTime,
+                        t.workItemId,
+                        t.formName,
+                        t.subject,
+                        a.taskId,
+                        a.status,
+                        a.activationTime,
+                        a.name,
+                        a.description,
+                        a.priority,
+                        a.actualOwner,
+                        a.createdBy,
+                        a.deploymentId,
+                        a.processId,
+                        a.processInstanceId,
+                        a.createdOn,
+                        a.dueDate,
+                        a.workItemId
+           FROM Task t
+           LEFT OUTER JOIN AuditTaskImpl a ON t.id = a.taskId
+           WHERE t.processInstanceId = :processInstanceId and t.status in ( :statuses)) task_union 
+        LEFT JOIN ProcessInstanceLog plog ON a_processInstanceId = plog.processInstanceId
     </query>
     <!-- hint name="org.hibernate.timeout" value="200"/ -->
-  </named-query>   
-
+  </named-native-query>
 
-  <!-- advance runtime queries -->
   <named-native-query name="GetProcessInstanceByIdList">
       <query>
         SELECT processInstanceId, processId, processName, processVersion, status, externalId, start_date, user_identity, correlationKey 
           FROM ProcessInstanceLog pil 
          WHERE processInstanceId IN (:idList) 
     </query>
   </named-native-query>
-  
+
   <named-native-query name="GetVariablesByProcessInstanceIdList">
       <query>
             SELECT A1.processInstanceId, A1.variableId, A1.value 
```

Comment:
Maybe the line numbers in RHPAM-4649 were incorrect (my bad, I should have double-checked), but the correct sequence order as per the xsd is named-query, named-native-query, sql-result-set-mapping. Therefore, please move <sql-result-set-mapping name="UserTaskInstanceDesc"> to the end of the file (after the last </named-native-query>). Thanks!